docs/book/tutorials/kubernetes-on-vsphere-with-kubeadm.md

```diff
@@ -1,58 +1,137 @@
-# Running a Kubernetes Cluster on vSphere with kubeadm
-
+# Deploying a Kubernetes Cluster on vSphere with CSI and CPI
+The purpose of this guide is to provide the reader with step by step instructions on how to deploy Kubernetes on vSphere infrastructure. The instructions use kubeadm, a tool built to provide best-practice “fast paths” for creating Kubernetes clusters. The reader will also learn how to deploy the Container Storage Interface and Cloud Provider Interface plugins for vSphere specific operations.
```

Comment:
We should wrap tool names and concepts in backticks to make sure they are highlighted - i.e. kubeadm -> `kubeadm`, PersistentVolume -> `PersistentVolume` - This goes for the document as a whole, not just the opening section. It helps clarify what is K8s parlance and what is just a phrase.

Reply:
Add something like "At the end of this tutorial you will have a fully running K8s on vSphere environment that allows for dynamic provisioning of volumes."

Reply:
Done.